src/astro/databases/google/bigquery.py

```diff
@@ -302,6 +302,49 @@ def get_project_id(self, target_table) -> str:
         except AttributeError:
             raise ValueError(f"conn_id {target_table.conn_id} has no project id")
 
+    def load_local_file_to_bigquery(
```

Comment:
Since we are within the `BigqueryDatabase`, perhaps  this should be named `load_local_file_to_table`

Reply:
Done.